dev/breeze/src/airflow_ci/selective_ci_utils.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
```

Comment:
we do not need "read_only" - it was only to get more sanity for BASH code, but using it in Python is not needed.